test/functional/combine_logs.py

```diff
@@ -19,22 +22,29 @@
 
 def main():
     """Main function. Parses args, reads the log files and renders them as text or html."""
-
-    parser = argparse.ArgumentParser(usage='%(prog)s [options] <test temporary directory>', description=__doc__)
+    parser = argparse.ArgumentParser(description=__doc__)
```

Comment:
nit: can you add `formatter_class=argparse.RawTextHelpFormatter` here. Otherwise the newlines from the docstring get squashed in the help text.